test/integration/authorization_test.go

```diff
@@ -445,10 +465,30 @@ func (test localResourceAccessReviewTest) run(t *testing.T) {
 			}
 		}
 
-		if actualResponse.Namespace != test.response.Namespace ||
-			!reflect.DeepEqual(actualResponse.Users.List(), test.response.Users.List()) ||
-			!reflect.DeepEqual(actualResponse.Groups.List(), test.response.Groups.List()) ||
-			actualResponse.EvaluationError != test.response.EvaluationError {
+		if actualResponse.Namespace != test.response.Namespace {
+			failMessage = fmt.Sprintf("%s\n: namespaces does not match (%s!=%s)", test.description, actualResponse.Namespace, test.response.Namespace)
+			return false, nil
+		}
+		if actualResponse.EvaluationError != test.response.EvaluationError {
+			failMessage = fmt.Sprintf("%s\n: evaluation errors does not match (%s!=%s)", test.description, actualResponse.EvaluationError, test.response.EvaluationError)
+			return false, nil
+		}
+
+		// FIXME: This should be already sorted by the server.
```

Comment:
@soltysh we should sort on server as well to provide consistent results to users (that is the point of that fixme).